### PR TITLE
LPS-115043 Empty Audit Graph when there is no content labeled with marketing categories

### PR DESCRIPTION
**Pull request by dgarciasarai**
Hey @bryceosterhaus! I talked to @jbalsas and it would be great that you can review the PRs about the Audit Graph chart. This chart will be hidden with a feature flag because we need another feature that is ready for dev and the Echo team is going to implement it in the near future.

This pull request contains the empty chart with a message and a button. Right now the button doesn't have any action and the message is implemented with `ClayEmptyState`.

empty-audit-graph

Also, I added a simple test to check the `EmptyBarChart` messages.

I have another technical task to make the chart responsive: https://issues.liferay.com/browse/LPS-115046. Any hint about how to do that? I'm going to try the `ResponsiveContainer` but I don't know if it will be enough (http://recharts.org/en-US/api/ResponsiveContainer).

Thanks!!!!

Feel free to send it to Brian once you review it and the CI passed.

cc: @wincent

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -11,3 +13,11 @@
 		}
 	}
 }
+
+.empty-state {
+	left: 39%;
```
Comment (jbalsas) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss:18]: Well... this seems oddly specific 😂

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyBarChart.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+			<ClayEmptyState
+				className="empty-state text-center"
+				description={Liferay.Language.get(
+					'please-create-marketing-categories-to-label-and-audit-your-content'
+				)}
+				title={Liferay.Language.get('no-data-yet')}
+			>
+				<ClayButton displayType="secondary">
+					{Liferay.Language.get('add-marketing-categories')}
+					<ClayIcon className="ml-1" symbol="shortcut" />
```
Comment (jbalsas) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyBarChart.js:43]: Not sure if `ClayButtonWithIcon` might help us get rid of this `ml-1` utility... 🤔

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss:18]: I didn't found something related with empty message in the chart out of the box. I have to position it manually. That makes me sad...

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyBarChart.js (hunk 2 above)
Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyBarChart.js:43]: It seems that `ClayButtonWithIcon` is not prepared to add a text along with the icon 🤔https://storybook.clayui.com/?path=/story/components-claybutton--buttonwithicon

These are the props for `ClayButtonWithIcon`:
```
interface IProps extends React.ComponentProps<typeof ClayButton> {
    spritemap?: string;
    symbol: string;
}
```

This is what I tried, but it doesn't work:
```
<ClayButtonWithIcon symbol="shortcut" displayType="secondary">
    Add Marketing Categories
</ClayButtonWithIcon>
```

Maybe this is something that we can change in Clay?

Comment (jbalsas) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyBarChart.js:43]: Well... having to add `ml-3` is indeed a smell that something's missing somewhere...

@pat270, @eduardoallegrini, @marcoscv-work?

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss:18]: FYI https://github.com/recharts/recharts/issues/1581

Comment (marcoscv-work) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss:18]: We will change this for a left 50% with translate -50%

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyBarChart.js (hunk 2 above)
Comment (marcoscv-work) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyBarChart.js:43]: We will use for this:
https://clayui.com/docs/css/utilities/inline-item.html
`inline-item inline-item-before`

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/css/main.scss]: https://github.com/bryceosterhaus/liferay-portal/pull/70/commits/93121106007125ca3a4747d3952dae739b0e184b

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyBarChart.js (hunk 2 above)
Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/EmptyBarChart.js]: https://github.com/bryceosterhaus/liferay-portal/pull/70/commits/71598d43eaf6bd97d2c4fd8b78b411f6d94e2f1c

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
Running the CI again because of two new commits (shouldn't affect...)

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Pull request closed without merging**